packages/cursorless-vscode-e2e/src/suite/fixtures/recorded/itemTextual/chuckEveryItem.yml

```diff
@@ -20,11 +20,12 @@ initialState:
       active: {line: 0, character: 1}
   marks: {}
 finalState:
-  documentContents: |-
-    {
-        
+  documentContents: |2-
+
```

Comment:
This happened because we didn't find a valid pair and fell back to the line being the iteration scope

Reply:
I'd prob update the test to have the cursor somewhere else so it works like intended